### PR TITLE
Release OrientDB v 2.2.31

### PR DESCRIPTION
**Pull request by luigidellaquila**

### CODE REVIEW

File: library/orientdb [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 2.1.25: git://github.com/orientechnologies/orientdb-docker@8a9633c19fa9c53a5446b9b62997ff389813e785 2.1
 
-2.2.30: git://github.com/orientechnologies/orientdb-docker@0a229bb2c91c9115023c5010a3b46e2cf5c24925 2.2/x86_64/alpine
+2.2.30: git://github.com/orientechnologies/orientdb-docker@98c76824843b8c44944a14c5bd62af7e358af670 2.2/x86_64/alpine
 
-2.2.30-spatial: git://github.com/orientechnologies/orientdb-docker@0a229bb2c91c9115023c5010a3b46e2cf5c24925 2.2-spatial/x86_64/alpine
+2.2.30-spatial: git://github.com/orientechnologies/orientdb-docker@98c76824843b8c44944a14c5bd62af7e358af670 2.2-spatial/x86_64/alpine
```
Comment (yosifkit) [inline library/orientdb]: The tags should also be updated: `2.2.30` and `2.2.30-spatial`.

Comment (luigidellaquila) [reply library/orientdb]: Sorry, my bad, I just pushed a fix

Thanks

Luigi

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index c1ec268..4f55e17 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,6 +1,6 @@
 orientdb:2.0.18 @ amd64
 orientdb:2.1.25 @ amd64
-orientdb:2.2.30-spatial @ amd64
+orientdb:2.2.31-spatial @ amd64
 orientdb:3.0.0m2 @ amd64
 orientdb:3.0.0m2-spatial @ amd64
 orientdb:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index ee16d05..817baaa 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,7 +1,7 @@
 orientdb:2.0.18
 orientdb:2.1.25
-orientdb:2.2.30
-orientdb:2.2.30-spatial
+orientdb:2.2.31
+orientdb:2.2.31-spatial
 orientdb:3.0.0m2
 orientdb:3.0.0m2-spatial
 orientdb:latest
diff --git a/orientdb_2.2.30-spatial/Dockerfile b/orientdb_2.2.31-spatial/Dockerfile
similarity index 80%
rename from orientdb_2.2.30-spatial/Dockerfile
rename to orientdb_2.2.31-spatial/Dockerfile
index 8f617fb..1391181 100644
--- a/orientdb_2.2.30-spatial/Dockerfile
+++ b/orientdb_2.2.31-spatial/Dockerfile
@@ -2,10 +2,10 @@
 # Dockerfile  for OrientDB with spatial module and neo4j connector
 ############################################################
 
-FROM orientdb:2.2.30
+FROM orientdb:2.2.31
 
-ENV ORIENTDB_DOWNLOAD_SPATIAL_MD5 52bd5e5da675a05db240babadde1c5c4
-ENV ORIENTDB_DOWNLOAD_SPATIAL_SHA1 0c36159eef6ad73ad07f8fc9a5dae056101638a8
+ENV ORIENTDB_DOWNLOAD_SPATIAL_MD5 45c20a72a89ea40a10b29064256b9856
+ENV ORIENTDB_DOWNLOAD_SPATIAL_SHA1 49ece1da0d4444bc231b7b8961c2f25c5e27d58b
 
 ENV ORIENTDB_DOWNLOAD_SPATIAL_URL ${ORIENTDB_DOWNLOAD_SERVER:-http://central.maven.org/maven2/com/orientechnologies}/orientdb-spatial/$ORIENTDB_VERSION/orientdb-spatial-$ORIENTDB_VERSION-dist.jar
 
diff --git a/orientdb_latest/Dockerfile b/orientdb_latest/Dockerfile
index deadb12..d8542b9 100644
--- a/orientdb_latest/Dockerfile
+++ b/orientdb_latest/Dockerfile
@@ -10,9 +10,9 @@ MAINTAINER OrientDB LTD (info@orientdb.com)
 #   docker build -t mine --build-arg ORIENTDB_DOWNLOAD_SERVER=http://repo1.maven.org/maven2/com/orientechnologies/ .
 ARG ORIENTDB_DOWNLOAD_SERVER
 
-ENV ORIENTDB_VERSION 2.2.30
-ENV ORIENTDB_DOWNLOAD_MD5 41757856ddbca87f368e812982041316
-ENV ORIENTDB_DOWNLOAD_SHA1 7c014647fbf7521a2817a5427e29dae609e53954
+ENV ORIENTDB_VERSION 2.2.31
+ENV ORIENTDB_DOWNLOAD_MD5 a7f9558bed6f8a12950b6282af9ef8cd
+ENV ORIENTDB_DOWNLOAD_SHA1 b4044e1a623ceaa67fd2a0ff18afe10b3f741c1c
 
 ENV ORIENTDB_DOWNLOAD_URL ${ORIENTDB_DOWNLOAD_SERVER:-http://central.maven.org/maven2/com/orientechnologies}/orientdb-community/$ORIENTDB_VERSION/orientdb-community-$ORIENTDB_VERSION.tar.gz
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #3797; b93a751a; `amd64` (`orientdb`):

```console

$ bashbrew build orientdb:2.0.18
Using bashbrew/cache:d872ee6e41183c17351073d7fc0b4be1e5e4ef3f84defb15444097098ef30eea (orientdb:2.0.18)
Tagging orientdb:2.0.18

$ test/run.sh orientdb:2.0.18
testing orientdb:2.0.18
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build orientdb:2.1.25
Building bashbrew/cache:a5b58da4e2bc3961c7f56d37dfbd91059c96970fd880766ffbe0289496b2c7ec (orientdb:2.1.25)
Tagging orientdb:2.1.25

$ test/run.sh orientdb:2.1.25
testing orientdb:2.1.25
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build orientdb:2.2.31
Building bashbrew/cache:ecce34533ed0ace7a7fff329b2fea94f714ca692c38b1a59f6a1952022808775 (orientdb:2.2.31)
Tagging orientdb:2.2.31
Tagging orientdb:latest

$ test/run.sh orientdb:2.2.31
testing orientdb:2.2.31
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build orientdb:2.2.31-spatial
Building bashbrew/cache:79f2b9fceef01861971b60d073932feaacf8a52594010178e9697100dac3fd21 (orientdb:2.2.31-spatial)
Tagging orientdb:2.2.31-spatial

$ test/run.sh orientdb:2.2.31-spatial
testing orientdb:2.2.31-spatial
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build orientdb:3.0.0m2
Building bashbrew/cache:5bf32400fa68771b16ec2d513ae3c8a8a0fc6fba71c693afa466746b7fdea1ab (orientdb:3.0.0m2)
Tagging orientdb:3.0.0m2

$ test/run.sh orientdb:3.0.0m2
testing orientdb:3.0.0m2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build orientdb:3.0.0m2-spatial
Building bashbrew/cache:92280293904a6d09e41f52013f9b3e3786d7c9c35336c22bea2d13e4560b85b7 (orientdb:3.0.0m2-spatial)
Tagging orientdb:3.0.0m2-spatial

$ test/run.sh orientdb:3.0.0m2-spatial
testing orientdb:3.0.0m2-spatial
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**